### ISSUE TITLE
Cluster event cleanup issue with MongoDB 2.2

### ISSUE THREAD

**Issue by bernd** [Graylog2/graylog2-server#1138]
Running with MongoDB 2.2.7, I see the following during server startup.

```
2015-05-04 17:32:15,411 WARN : org.graylog2.events.ClusterEventCleanupPeriodical - Error while removing stale cluster events from MongoDB
com.mongodb.WriteConcernException: { "serverUsed" : "127.0.0.1:27017" , "n" : 0 , "connectionId" : 3 , "wnote" : "no replication has been enabled, so w=2+ won't work" , "err" : "norepl" , "ok" : 1.0}
        at com.mongodb.CommandResult.getWriteException(CommandResult.java:90)
        at com.mongodb.CommandResult.getException(CommandResult.java:79)
        at com.mongodb.CommandResult.throwOnError(CommandResult.java:140)
        at com.mongodb.DBTCPConnector._checkWriteError(DBTCPConnector.java:135)
        at com.mongodb.DBTCPConnector.access$000(DBTCPConnector.java:40)
        at com.mongodb.DBTCPConnector$1.execute(DBTCPConnector.java:186)
        at com.mongodb.DBTCPConnector$1.execute(DBTCPConnector.java:181)
        at com.mongodb.DBPort.doOperation(DBPort.java:187)
        at com.mongodb.DBTCPConnector.doOperation(DBTCPConnector.java:208)
        at com.mongodb.DBTCPConnector.say(DBTCPConnector.java:181)
        at com.mongodb.DBCollectionImpl.remove(DBCollectionImpl.java:232)
        at com.mongodb.DBCollectionImpl.remove(DBCollectionImpl.java:203)
        at com.mongodb.DBCollection.remove(DBCollection.java:300)
        at org.mongojack.JacksonDBCollection.remove(JacksonDBCollection.java:726)
        at org.graylog2.events.ClusterEventCleanupPeriodical.doRun(ClusterEventCleanupPeriodical.java:107)
        at org.graylog2.plugin.periodical.Periodical.run(Periodical.java:83)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
        at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:304)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:178)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:745)
```

Might be related: http://docs.mongodb.org/manual/reference/write-concern/ ("majority")

**Comment by kroepke** [comment]
referenced in #1137

**Comment by bernd** [comment]
Fixed in 01fee82.
